resolvconf/dns/resolvconf_test.go

```diff
@@ -0,0 +1,39 @@
+package dns
+
+import (
+	"strconv"
+	"testing"
+)
+
```

Comment:
try to add this: `var _ = flag.Bool("incontainer", false, "Indicates if the test is running in a container")`

Reply:
Why does add this? I don't understand.